tensorflow/compiler/xla/metric_table_report.cc

```diff
@@ -80,9 +80,11 @@ void MetricTableReport::WriteReportToInfoLog(double expected_metric_sum) {
 
   int64 pos = 0;
   const string report = MakeReport(expected_metric_sum);
-  while (pos < report.size()) {
+  int report_size = report.size();
```

Comment:
Let's try to make all of these new vars `const`. Here and throughout the PR.

Reply:
gotcha!